modules/hostap/src/supp_api.h

```diff
@@ -19,6 +19,17 @@
 #define MAC_STR_LEN 18 /* for ':' or '-' separated MAC address string */
 #define CHAN_NUM_LEN 6 /* for space-separated channel numbers string */
 
+#ifdef CONFIG_WIFI_NM_WPA_SUPPLICANT_CRYPTO_ENTERPRISE
+struct wpa_supp_enterprise_file {
```

Comment:
how are these taken as input from the user? Is that support coming in upcoming PRs?

Reply:
it would be better to add this to Wi-Fi shell so that we can test/see the usage of the API.

Reply:
Vendor can decide when to set the enterprise file.
supplicant_add_enterprise_file is called in NXP wifi code.

Reply:
Hi @krish2718 , can I only add the caller of supplicant_add_enterprise_file into wifi_mgmt.c, not the wifi_shell.c?
as I find input the shell to add enterprise file is weird.

Reply:
> as I find input the shell to add enterprise file is weird.

Can you please elaborate? `wifi pmksa_flush` command should be fine right?

Reply:
okay, wifi pmksa_flush, I will add to wifi shell